service/controller/v15/resource/deployment/master_deployment.go

```diff
@@ -325,6 +325,11 @@ func newMasterDeployments(customObject v1alpha1.KVMConfig) ([]*extensionsv1.Depl
 								Name:            "shutdown-deferrer",
 								Image:           key.ShutdownDeferrerDocker,
 								ImagePullPolicy: apiv1.PullAlways,
+								Command: []string{
```

Comment:
Can't we use args or something? The entrypoint should already be the binary and daemon command.